.github/workflows/codespell.yml

```diff
@@ -0,0 +1,18 @@
+# GitHub Action to automate the identification of common misspellings in text files
+# https://github.com/codespell-project/codespell
+name: codespell
+on: [push, pull_request]
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install codespell flake8
+      - run: |
+          SKIP="./bin/trace,./hl/tools/h5watch/h5watch.c,./tools/test/h5jam/tellub.c,./.git"
+          IGNORE="isnt,inout,nd,parms,parm,ba,offsetP,ser,ois,had,fiter,fo,clude,refere,minnum,offsetp,creat,ans:,eiter,lastr,minum,ans,isn't,ifset,sur,trun,dne,tthe,hda,filname,te,htmp"
+          codespell -L $IGNORE --skip=$SKIP --quiet-level=2
+      - name: Codespell comment
+        if: ${{ failure() }}
+        uses: plettich/python_codespell_action@master
```

Comment:
Just use:
https://github.com/codespell-project/actions-codespell/

Although you'll need to make some minor tweaks to your config. It'll annotate rather than commenting (so they vanish when they're stale.

Reply:
Thanks, I updated to use this instead. 

Great tool BTW, it saved us a ton of time, and as you can tell it was surely needed. We had only a few glitches.